### PR TITLE
[plugin.video.on_aol] 1.1.0

### PR DESCRIPTION
**Pull request by divingmule**
A long overdue update for this plugin that is based off of on.aol.com.

Thanks!

### CODE REVIEW

**Comment by learningit** [comment]
Looks good to me

File: plugin.video.on_aol/addon.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.beautifulsoup" version="3.2.1"/>
+    <import addon="script.common.plugin.cache" version="2.5.6"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary>Videos form on.aol.com.</summary>
```
Comment (phil65) [inline plugin.video.on_aol/addon.xml]: typo ("from")

**Comment by phil65** [comment]
One typo in addon.xml, code looks fine.

**Comment by divingmule** [comment]
Nice catch @phil65. Fixed typo > 84227ab

**Comment by phil65** [comment]
Just as info for next version : https://github.com/alanwww1/xbmc-xml2po/downloads

**Pull request merged**